docs/getting-started.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Getting Started
+nav: 0.1
+---
+Welcome to the Jotai Getting Started tutorial!
+This tutorial will introduce you to Jotai and teach you how to start using it correctly.
+
+For this tutorial, we assume that you're using Jotai with React. The example is based on [a Create-React-App folder structure](https://create-react-app.dev/docs/folder-structure) so the first thing you have to do is:
```

Comment:
Do you think CRA is better than web based environment, like codesandbox or stackblitz?
I'm not sure if we should mention about the folder structure.

Reply:
I think CRA is more common for explaining the concept and we can use codesandbox at the bottom as a demo for readers to interact with